source/includes/_event_triggered_survey.md

```diff
@@ -0,0 +1,34 @@
+#Event Triggered Survey
+
+```javascript
+// You define a global variable to track your events
+my_global_event_name = 'loaded_website'; 
+
+// You define your WootricSettings variable at the beginning of the code
+wootricSettings = {
+  account_token: 'NPS-YOURTOKEN',
+  email: 'user@example.com',
+  created_at: 1528416000,
+  properties: {
+    favorite_color: 'blue'
+  },
+  event_name: my_event
+};
+
+// You assign the event name when a given action happens
+wootricSettings.event_name = 'on_purchase'
+
+// You call Wootric whenever you want to check if the event you set might be eligible for a survey
+wootric('run');
+```
+
+When you install Wootric via our Segment integrations events are automatically passed to us and you can trigger surveys based on those. But what if you installed Wootric natively?
+
+This article will guide you through the implementation process so you can get your surveys up and running!
```

Comment:
Replace this line with:

This article will help you to set up your Wootric Targeted Sampling Surveys using a native Wootric installation.